public/vue-chat/app.js

```diff
@@ -0,0 +1,160 @@
+// A placeholder image if the user does not have one
+const PLACEHOLDER = '/placeholder.png';
```

Comment:
Ah shoot. I guess the only thing we need to do is change this to the new placeholder image.